turborepo-tests/integration/tests/turbo-help.t

```diff
@@ -263,6 +263,7 @@ Test help flag for login command
   Options:
         --sso-team <SSO_TEAM>             
         --version                         
+    -f  --force
```

Comment:
Can we add a description here? Something like: "Overwrite any existing turbo or vercel tokens"

Reply:
assuming the following is correct, how about "Force a login to receive a new token. Will overwrite any existing tokens for the given login url".

This is not a vercel-specific feature, it should work with any backend.